### PR TITLE
Fix/search ui

### LINKED ISSUE #35103: Improve search UI
Please see below image with suggested improvements:

echo_and_print___freecodecamp_guide

### PR DESCRIPTION
**Pull request by ahmaxed**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35103

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
Thanks for the review @thecodingaviator.

File: client/src/components/Header/header.css [hunk 1]
```diff
@@ -118,6 +118,16 @@ li.user-state-link > a:focus {
     left: 10%;
   }
 }
+/* hits */
+.ais-Highlight-highlighted {
```
Comment (ValeraS) [inline client/src/components/Header/header.css]: Try something more specific to remove `!important`
```suggestion
.fcc_searchBar .ais-Highlight-highlighted {
```

**Comment by ahmadabdolsaheb** [comment]
Thanks @ValeraS for the help. I resolved it in the way you suggested. Also added media queries to the suggestions wrapper.

**Comment by ahmadabdolsaheb** [comment]
@raisedadead @ValeraS this one is ready for Review/merge. screenshots are included in the issue.

**Review by ValeraS — DISMISSED** [review]

**Comment by ahmadabdolsaheb** [comment]
@raisedadead @ValeraS, 🙏

File: client/src/components/Header/header.css [hunk 2]
```diff
@@ -105,16 +105,45 @@ header {
...
+    position: relative;
+  }
+  .fcc_searchBar .ais-Hits {
+    width: calc(100% - 20px);
+    left: 10px;
+  }
+}
+
+/* hits */
+.ais-Hits-item .ais-Highlight-highlighted {
```
Comment (ValeraS) [inline client/src/components/Header/header.css]:
```suggestion
.fcc_searchBar .ais-Highlight-highlighted {
```

File: client/src/components/Header/header.css [hunk 3]
```diff
@@ -105,16 +105,45 @@ header {
...
+  background-color: transparent;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.ais-Hits-item .fcc_suggestion_item {
```
Comment (ValeraS) [inline client/src/components/Header/header.css]:
```suggestion
.fcc_searchBar .fcc_suggestion_item {
```

**Review by ValeraS — CHANGES_REQUESTED** [review]

**Review by ValeraS — APPROVED** [review]

**Pull request merged**